applications/plugins/SofaSphFluid/SPHFluidSurfaceMapping.cpp

```diff
@@ -39,15 +39,10 @@ using namespace sofa::defaulttype;
 // Register in the Factory
 int SPHFluidSurfaceMappingClass = core::RegisterObject("TODO-SPHFluidSurfaceMappingClass")
```

Comment:
Maybe the chance to update the doc..?

Reply:
Lol I was thinking the same but didn't had the courage to say it :)

Reply:
Sorry but removing a TODO to leave just the name in the docstring is not what I call an update. Please provide some details on what do the object ;)